platform_usermanagement/index.js

```diff
@@ -285,6 +285,10 @@ function updatePassword(cognitoClient, config, userData) {
 	});
 }
 
+function createUserInSCM(config, userData) {
```

Comment:
Shouldn't line 114 (https://github.com/tmobile/jazz/pull/70/files#diff-e8b91dbd741ea45e02e7efea12ad8ef9L114) be updated to say createUserInSCM?

Reply:
Supposed to change in 114. Changed it

Reply:
I don't see this change, line 114 still says: 

`.then((s) => rp(createUserInBitBucket(config, service_data, s.UserSub)))`